docs/configuration.md

```diff
@@ -131,7 +131,8 @@ spec:
 | `sans`                       | List of additional addresses to push to API servers serving the certificate.                                                                                                                                                                                              |
 | `ca.expiresAfter`            | The expiration duration of the CA certificate (default: 87600h)                                                                                                                                                                                                           |
 | `ca.certificatesExpireAfter` | The expiration duration of the server certificate (default: 8760h)                                                                                                                                                                                                        |
-| `extraArgs`                  | Map of key-values (strings) for any extra arguments to pass down to Kubernetes API server process. Any behavior triggered by these parameters is outside k0s support.                                                                                                     |
+| `extraArgs`                  | Map of key-values (strings) for any extra arguments to pass down to Kubernetes API server process. `extraArgs` are recommended over `rawArgs` if the use case allows it. Any behavior triggered by these parameters is outside k0s support. (default: empty)              |
```

Comment:
I'd document the "`extraArgs` are recommended over `rawArgs`" stuff on rawArgs, not on extraArgs.